packages/discord.js/src/util/Util.ts

```diff
@@ -558,12 +593,20 @@ class Util extends null {
    * @param {string} text The string to be converted
    * @returns {string}
    */
-  static cleanCodeBlockContent(text) {
+  public static cleanCodeBlockContent(text: string) {
     return text.replaceAll('```', '`\u200b``');
   }
 }
 
-module.exports = Util;
-
 // Fixes Circular
-const GuildChannel = require('../structures/GuildChannel');
+import GuildChannel from '../structures/GuildChannel';
+import type {
+  AnyChannel,
+  ColorResolvable,
+  EmojiIdentifierResolvable,
+  GuildTextBasedChannel,
+  MakeErrorOptions,
+  TextBasedChannel,
+} from '../../typings';
+import type { Role } from '../structures/Role';
+import type Client from '../client/Client';
```

Comment:
Those don't need to go at the bottom, since they're type imports.

Reply:
Blame VSC Autoimport :P